docs/tutorials/interfaces.rst

```diff
@@ -0,0 +1,52 @@
+.. _interfaces:
+========================
+Interfaces
+========================
+
+
+One of the biggest strengths of Orquestra is its modularity. Integrating new backends, optimizers, ansatzes, etc. does not require changing the core code - it requires only creating a new module that conforms to existing interfaces and therefore can be used across the whole platform. These interfaces allow us to implement various modules and functions.
+
+Benefits of interfaces
+~~~~~~~~~~~~~~~~~~~~~~
+
+Interface offers the ability to define behaviour that can be implemented by unrelated classes without forcing them to share a common class hierarchy. This results in code flexibility and faster development with easier testing options. 
```

Comment:
Other benefits, for now add them just as bullets:
- Interoperability
- Modularity
- Dependencies decoupling 

I think we could link this: https://www.zapatacomputing.com/we-felt-your-pain-and-built-orquestra/, as relevant read that gives more context

Reply:
Done